[Documentation] Add main section blocks in homepage and widget examples (#1419)
GeotrekCE/Geotrek-rando-v3: docs/customization/customization-htmlscripts.md

When choosing a template name, care must be taken not to select a reserved name used by sections defined by the application (e.g `presentation`, see [example](https://github.com/GeotrekCE/Geotrek-rando-v3/blob/main/frontend/config/details.json)).
If you do, the customized template will not be displayed.

**Step 2:** Copy the template name without the `.html` suffix into the `customization/html/details.json` file.
For example I want to display it in treks and outdoor sites details page:

```json
{
  "sections": {
    "trek": [
      {
        "name": "example",
        "display": true,
        "anchor": true,
        "order": 11
      }
    ],
    "outdoorSite": [
      {
        "name": "example",
        "display": true,
        "anchor": true,
        "order": 11
      }
    ]
  }
}
```

**Step 3:**. Copy the section title/anchor into the translations files.
For example in `customization/translations/en.json`:

```json
{
  "details": {
    "example": "My example"
  }
}
```

#### Widgets examples

You can embed third-party widgets into Geotrek-rando detail pages by adding their HTML code in the appropriate **`customization.html/details/`** subfolder.

> The widget can exploit data from the geotrek page to customize the displayed information (departure city, trek geometry, trek name, etc.).
> 
> Here is some examples but there is a lot more that can be done, your imagination is your limit !

Reviewer: ```suggestion
The widget can [use data from the Geotrek page](#templates-available-on-home-page) to customize the information displayed (departure city, trek geometry, trek name, etc.).

Here are a few examples, but there is much more you can do, your imagination is the limit!
```

}
```

#### Widgets

You can embed third-party widgets into Geotrek-rando detail pages by adding their HTML code in the appropriate **`customization.html/details/`** subfolder.

##### **Weather Forecast (Météo-France)**

![Search](../img/widget-meteofrance.png)

**File location:**
`customization.html/details/forecastWidget.html`

**Project page:** [Météo-France Widgets](https://meteofrance.com/widgets)

**Example code:**

```html
<iframe
  id="widget_autocomplete_preview"
  loading="lazy"
  class="w-full"
  height="150"
  src="https://meteofrance.com/widget/prevision/{{ cityCode }}0"
  title="Météo-France Widget">
</iframe>
```

##### **Air Quality (Atmo)**

![Search](../img/widget-atmo.png)

**File location:**
`customization.html/details/atmos.html`

**Project page:** [Atmo Widget](https://www.atmo-hdf.fr/widget)

**Example code:**

```html
<div style="height:470px;">
  <a href="https://www.atmo-hdf.fr/">
    <iframe
      id="widget_atmo"
      loading="lazy"
      src="https://www.atmo-hdf.fr/widget-mon-air/widget/commune/{{ cityCode }}"
      class="w-full h-full b-0"
      title="Atmo Widget">
    </iframe>
  </a>
</div>
```

> 

Reviewer: The section page call is missing.

> 
> First, you need to create a file with the HTML content.
> Then, you can call it in the section pages by selecting the file name without the `.html` extension.
> For example, if you want to display the `atmos` widget defined above in the trek pages just after its description you have to edit `/customization/config/details.json` file:
> 
> ```json
> {
>   "sections": {
>     "trek": [
>       {
>         "name": "atmos",
>         "display": true,
>         "anchor": false,
>         "order": 45
>       },
>     ]
> }
> ```

}
```

##### **Air Quality (Atmo)**

![air-quality-widget](../img/widget-atmo.png)

**Configuration** [Check this comment](https://github.com/GeotrekCE/Geotrek-rando-v3/issues/914#issuecomment-1779758757)

##### **Sustainable transport route planner**

> Multiple widgets can be added to help citizens plan their trip using sustainable transports. Here is some examples but we advise you to look for local solutions, there is a good chance your country / state / city may have an existing widget that you can integrate on your website.

Reviewer: ```suggestion
Multiple widgets can be added to help users to plan their trip using sustainable transports. Here are a few examples but we advise you to look for local solutions, there is a good chance your country / state / city may have an existing widget that you can integrate on your website.
```
